extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -58,6 +59,10 @@ public function render( $markup, Parser $parser, PPFrame $frame, $params = null
 		if ( !isset( $params ) ) {
 			$params = $infoboxParser->getInfoboxParams( $markup );
 		}
+
+		$infoboxAttributesValidator = new Wikia\PortableInfobox\Helpers\InfoboxTagAttrValidator();
+		$infoboxAttributesValidator->validateAttributes( $params );
```

Comment:
A nitpick: we use $params as the attribute names. Maybe it'll read more fluently if the naming was consistent